app/src/main/kotlin/compiler/syntax/LanguageTokens.kt

```diff
@@ -34,6 +36,10 @@ object LanguageTokens {
             Pair(TokenType.FOREIGN2, "zewnętrzny"),
             Pair(TokenType.AS, "jako"),
 
+            // Array related keywords
+            Pair(TokenType.LENGTH, "długość"),
+            Pair(TokenType.ALLOCATE, "alokacja"),
```

Comment:
Are you sure that "alokacja" is the best choice here? While it is a Polish word, it's a loan translation from English "allocation" while we tend to prefer words of more Polish origin (look for example at "czynność" instead of "funkcja").

Reply:
Yeah, I wasn't so sure about it myself. I used it to match the string representation in ASTNode. I think "przydział" is the best Polish alternative. We can also use word not related to "allocate" but maybe "new", "create", "array" or "table". I need some outside opinion on that.

Reply:
Maybe just "tablica"?

Reply:
Or "ciąg" if we prefer unusual keywords.

Reply:
It could also be "rząd", "szyk" (this one technically has German etymology) or "szereg" (from Hungarian).